packages/ariakit-core/src/form/form-store.ts

```diff
@@ -18,15 +18,10 @@ import {
   setup,
   throwOnConflictingProps,
 } from "../utils/store.js";
-import type {
-  AnyObject,
-  PickRequired,
-  SetState,
-  SetStateAction,
-} from "../utils/types.js";
+import type { PickRequired, SetState, SetStateAction } from "../utils/types.js";
 import type { DeepMap, DeepPartial, Names, StringLike } from "./types.js";
 
-type Values = AnyObject;
+type Values = Record<string, unknown>;
```

Comment:
I'm thinking now that this could represent a breaking change since this `Values` type is exposed as `FormStoreValues` below.

We can do that in a separate PR, but it's probably safer for this PR to just include the `Array.isArray` fix.

Reply:
Good catch. I'll remove these changes from my PR.